stubs/tensorflow/tensorflow/keras/layers/__init__.pyi

```diff
@@ -127,7 +129,7 @@ class Dense(Layer[tf.Tensor, tf.Tensor]):
         kernel_constraint: _Constraint = None,
         bias_constraint: _Constraint = None,
         trainable: bool = True,
-        dtype: _LayerDtype = None,
```

Comment:
One optional improvement, there's comment near here that some of these signatures can be simplified with Unpack TypedDict ([pep 692](https://peps.python.org/pep-0692/)). The upstream stubs actually do use that technique which saves on repetition of dtype/trainable/name/dynamic/etc. I see that [mypy](https://github.com/python/typeshed/issues/9710) is checked in this issue. 

Can we start using 692 in typeshed now (has it been used yet?) @JelleZijlstra

Reply:
Still waiting for pytype as you can see in #9710